bedevere/bpo.py

```diff
@@ -37,6 +44,17 @@
         else:
             status = FAILURE_STATUS
     else:
+        if "body" in event.data["pull_request"]:
+            body = event.data["pull_request"]["body"]
+            if CLOSING_TAG not in body:
+                issue_number = issue_number_found.group("issue")
+                BPO_MSG = f"""{body}\n
```

Comment:
Might as well just have an actual newline in the triple-quoted string rather than a newline literal (I actually overlooked it when I was checking that there was a newline separating the body from the link).